lego/apps/flatpages/tests/test_api.py

```diff
@@ -52,7 +77,39 @@ def test_unauthorized(self):
     def test_create_page(self):
         page = {"title": "cat", "content": "hei"}
 
-        user = create_super_user()
-        self.client.force_authenticate(user)
+        superuser = create_super_user()
```

Comment:
Should test with `/sudo/admin/pages/create/` and/or `sudo/admin/pages/` instead.